homeassistant/components/fyta/coordinator.py

```diff
@@ -48,7 +49,12 @@ async def _async_update_data(
         ):
             await self.renew_authentication()
 
-        return await self.fyta.update_all_plants()
+        try:
+            data = await self.fyta.update_all_plants()
+        except (FytaConnectionError, FytaPlantError) as err:
+            raise UpdateFailed(err) from err
+
+        return data
```

Comment:
```suggestion
            return await self.fyta.update_all_plants()
        except (FytaConnectionError, FytaPlantError) as err:
            raise UpdateFailed(err) from err
```
This would also work right